README.md

```diff
@@ -1365,6 +1365,18 @@ Default: `0`
 
     let g:ycm_csharp_server_port = 0
 
+### The `g:ycm_csharp_insert_namespace_expr` option
```

Comment:
You might want to add a bit of text explaining why this option might be useful to people.